### PR TITLE
Ryan/tsne

### PR DESCRIPTION
**Pull request by Ryan-Rhys**
- Added scripts for UMAP and t-SNE
- Included example images for the ice-phases example

### CODE REVIEW

**Comment by Ryan-Rhys** [comment]
I don't have the subset.xyz file locally for the ice-phases example. If we want to keep this could someone send it to me? Having trouble downloading it from the repo.

**Comment by BingqingCheng** [comment]
You mean the subset.xyz for singlet-triplet-fission? That has been deleted.

**Comment by BingqingCheng** [comment]
All look good to me! Many thanks Ryan!

File: examples/ice-phases/README.md [hunk 1]
```diff
@@ -5,8 +5,16 @@ Nature Communicationsvolume 9, Article number: 2173 (2018)
 #
 * step 1
 
-gen_soap_descriptors.py -fxyz ice-dataset.xyz --rcut 4 --n 4 --l 6 --g 0.2 --periodic True --output xyz
+python ../../scripts/gen_soap_descriptors.py -fxyz ice-dataset.xyz --rcut 4 --n 4 --l 6 --g 0.2 --periodic True --output xyz
 
 * step 2
 
-pca.py -fxyz ASAP-n4-l6-c4.0-g0.2.xyz -fmat SOAP-n4-l6-c4.0-g0.2 -color ice-properties.dat --colorscolumn 3
+python ../../scripts/pca.py -fxyz ASAP-n4-l6-c4.0-g0.2.xyz -fmat SOAP-n4-l6-c4.0-g0.2 -color ice-properties.dat --colorscolumn 3
```
Comment (stenczelt) [inline examples/ice-phases/README.md:12]: why is there a need to use the relative path? The scripts are installed as well with everything else

File: examples/ice-phases/README.md [hunk 2]
```diff
@@ -5,8 +5,16 @@ Nature Communicationsvolume 9, Article number: 2173 (2018)
 #
 * step 1
 
-gen_soap_descriptors.py -fxyz ice-dataset.xyz --rcut 4 --n 4 --l 6 --g 0.2 --periodic True --output xyz
+python ../../scripts/gen_soap_descriptors.py -fxyz ice-dataset.xyz --rcut 4 --n 4 --l 6 --g 0.2 --periodic True --output xyz
```
Comment (stenczelt) [inline examples/ice-phases/README.md:8]: same with the relative path

**Comment by stenczelt** [comment]
Good job @Ryan-Rhys!

**Pull request merged**